### ISSUE TITLE
My Account edit-address endpoint text is missing "Addresses".

### ISSUE THREAD

**Issue by matanmn33** [woothemes/woocommerce#36490]
### Prerequisites

- [X] I have carried out troubleshooting steps and I believe I have found a bug.
- [X] I have searched for similar bugs in both open and closed issues and cannot find a duplicate.

### Describe the bug

As we discussed in the WooCommerce WordPress Forums here: https://wordpress.org/support/topic/my-account-endpoint-text-addresses-in-turkish-is-missing/#post-16385202
There you can find out about this bug.

### Expected behavior

"Addresses" text from My Account 'edit-address' endpoint should show itself.

### Actual behavior

"Addresses" text from My Account 'edit-address' endpoint is missing.

### Steps to reproduce

https://prnt.sc/Yc3jj1pH3c4D Screenshot in Hebrew, as well in the forum Turkish and France same issue.
I tried everything as a front-end developer and full-stack co-developer.
Searched conflicts in themes and plugins as well as woocommerce plugin folder.

### WordPress Environment

Already rolled back version so i don't have this thing there.

### Isolating the problem

- [X] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [X] This bug happens with a default WordPress theme active, or [Storefront](https://woocommerce.com/storefront/).
- [X] I can reproduce this bug consistently using the steps above.

**Comment by raifd** [comment]
5869342-zen
5872834-zen
5875973-zen

**Comment by Babylon1999** [comment]
4 more reports in forums:

- https://wordpress.org/support/topic/account-display-issue-after-update-woocommerce/#post-16379379
- https://wordpress.org/support/topic/edit-adress-not-showing-in-my-account/
- https://wordpress.org/support/topic/title-adresse-is-missing/
- https://wordpress.org/support/topic/translation-in-account-dashboard/ 

I was not able to reproduce the issue even on a fresh install but it seems to be affecting translated sites.

**Comment by devvnawaz** [comment]
Another report here: 5873805-zen

**Comment by mredodos** [comment]
Same problem with Italian translation of WooCommerce 

It seems the problem is here in the code wc-account-functions.php line 102 in function wc_get_account_menu_items() the __n function for plural it seems not work properly.

**Comment by swatipawarGS** [comment]
Hi @matanmn33,

Thank you for taking the time to report this bug, we really appreciate your help. We are able to reproduce the issue on our end for Hebrew language using  WooCommerce 7.3.0 version and WordPress 6.1.1 version. 

Please find below the screenshot for reference : 

![#36490](https://user-images.githubusercontent.com/94531721/213373322-da6d5d04-6f23-45c1-8281-00a0db772ad9.jpg)


Priority of this issue will be assigned shortly after we’d go through a round of prioritization. Please note it may take a couple of days.

**Comment by anastas10s-afk** [comment]
5873667-zen

**Comment by sertal70** [comment]
+1

Wordpress 6.1.1
Woocommerce 7.3.0
Using Italian language with Loco Translate plugin v2.6.3

**Comment by Kevinlafab** [comment]
+1

**Comment by RossThinkZap** [comment]
+1

UK Translation missing plural:

#: includes/admin/class-wc-admin-setup-wizard.php:511
#: includes/wc-account-functions.php:102
#: packages/woocommerce-blocks/src/StoreApi/Schemas/V1/AbstractAddressSchema.php:37
#: packages/woocommerce-blocks/src/StoreApi/Utilities/OrderController.php:330
#: assets/client/admin/app/index.js:2
#: packages/woocommerce-blocks/build/wc-settings.js:1
msgid "Address"
msgid_plural "Addresses"
msgstr[0] "Address"
msgstr[1] ""

**Comment by xue28** [comment]
https://wordpress.org/support/topic/bug-on-customer-profile-menu-address-link/#post-16399933

**Comment by xue28** [comment]
https://wordpress.org/support/topic/addresses-menu-and-addresses-dashboard-tab/#post-16400381

**Comment by anastas10s-afk** [comment]
5889753-zen

**Comment by harkalygergo** [comment]
Hungarian issue: https://wordpress.org/support/topic/addresses-text-on-account-page/

**Comment by zoupkat** [comment]
Another issue here (Italy): 5893214-ZD

**Comment by xue28** [comment]
Slovenian issue: https://wordpress.org/support/topic/edit-addresses-missing/?view=all#post-16403476

**Comment by raifd** [comment]
5880132-zen

**Comment by xue28** [comment]
https://wordpress.org/support/topic/error-addresses/#post-16406965

**Comment by jtobiesen** [comment]
5893672-zen

**Comment by igorhereira** [comment]
5873667-zen

**Comment by Dan662** [comment]
Same with UK:

https://wordpress.org/support/topic/missing-address-in-menu-delete-language-reappears-but-language-keeps-updating/

**Comment by anastas10s-afk** [comment]
5909800-zen

**Comment by xue28** [comment]
https://wordpress.org/support/topic/addresses-in-my-account-is-impossible-to-translate/

**Comment by xue28** [comment]
Italian issue: https://wordpress.org/support/topic/woocommerce-my-account-missing-plural-translation/

**Comment by EdoardoVignati** [comment]
Is this the line with the error? Can someone else please confirm?

https://github.com/woocommerce/woocommerce/blame/5d0c82f5c3982b6cf209a46f4af180387ece7964/plugins/woocommerce/includes/wc-account-functions.php#L102

**Comment by harkalygergo** [comment]
> Is this the line with the error? Can someone else please confirm?
> 
> https://github.com/woocommerce/woocommerce/blame/5d0c82f5c3982b6cf209a46f4af180387ece7964/plugins/woocommerce/includes/wc-account-functions.php#L102

@EdoardoVignati  I linked the same previously: https://wordpress.org/support/topic/addresses-text-on-account-page/

**Comment by mredodos** [comment]
> Is this the line with the error? Can someone else please confirm?
> 
> https://github.com/woocommerce/woocommerce/blame/5d0c82f5c3982b6cf209a46f4af180387ece7964/plugins/woocommerce/includes/wc-account-functions.php#L102

yes it is

**Comment by Maarten112** [comment]
+1, found the same issue today when staging with a default en_gb language. 

~~The problem is in includes/wc-account-functions on line 102~~

~~Old line (102):~~
~~'edit-address'    => _n( 'Address**es**', 'Address', (int) wc_shipping_enabled(), 'woocommerce' ),~~

~~New line (102):~~ 
~~'edit-address'    => _n( 'Address', 'Address**es**', ( 1 + (int) wc_shipping_enabled() ), 'woocommerce' ),~~

~~When changing new line (102) to:
`'edit-address'    => _n( 'Addresses', 'Address', ( 1 + (int) wc_shipping_enabled() ), 'woocommerce' ),`~~

~~It seems to function again. ~~

Edit 4/2/203:
It's actually pretty easy; the translation files are simply missing the plural translation :-) 
You can use the official translation page (https://translate.wordpress.org/locale/en-gb/default/wp-plugins/woocommerce/) or glotpress/any poeditor you like.
![Schermafbeelding 2023-02-04 163932](https://user-images.githubusercontent.com/10561736/216776330-ccb6c13d-b553-4325-abe2-e28f6ce704fc.png)

**Comment by Kevinlafab** [comment]
Hello do you know when can we have a woocommerce update to reseolve this issue ?
thanks :)
kevin

**Comment by munford** [comment]
In /wp-content/plugins/woocommerce/includes/wc-account-functions.php I changed line 102 from: 
`'edit-address'    => _n( 'Address', 'Addresses', ( 1 + (int) wc_shipping_enabled() ), 'woocommerce' ),`
to:
`'edit-address'    => _n( 'Addresses', 'Address', ( 1 + (int) wc_shipping_enabled() ), 'woocommerce' ),`
and it now works fine but I can't find where to add a Danish translation for the "Address" and also for "My Subscription"?

**Comment by harkalygergo** [comment]
@munford , yes, it would work like this, but it is not valid, `_n()` function first parameter has to be the single, second the plural version. [https://developer.wordpress.org/reference/functions/_n/](https://developer.wordpress.org/reference/functions/_n/)

**Comment by munford** [comment]
> 

OK I will wait for a real fix

**Comment by Maarten112** [comment]
> Hello do you know when can we have a woocommerce update to reseolve this issue ? thanks :) kevin

I already made the translation for the en_GB version, see https://translate.wordpress.org/projects/wp-plugins/woocommerce/stable/en-gb/default/?filters%5Bstatus%5D=either&filters%5Boriginal_id%5D=578237&filters%5Btranslation_id%5D=100864283

But it needs to be approved and pushed. I don't know how often this is done. 

My own temporary solution is to manually edit my woocommerce_en_GB.po file to include the plural of Address and build the mo file by that one. Thereafter, upload it to your languages folder. It will get overwritten by the first update that way, but you can also make it permanently by uploading to a different folder (however I don't recommend that). 

There are other options too, like only letting wordpress overwrite that single translation. Do a quick google search and you can find your options.

**Comment by anastas10s-afk** [comment]
5931729-zen 
**Portuguese (BR)**

**Comment by lgdelai** [comment]
Same Here: Language Portuguese (BR)

https://wordpress.org/support/topic/address-option-is-not-displayed-in-the-my-account-menu/#post-16448808

Does anyone have any idea when this will be resolved?

**Comment by xue28** [comment]
UK Language: https://wordpress.org/support/topic/my-account-addresses-is-missing-in-my-account-page/

**Comment by xue28** [comment]
https://wordpress.org/support/topic/bug-with-the-translation-of-this-word/

**Comment by EdithAllison** [comment]
WooCommerce 7.4.0 continues to have this bug, tested with en_GB. I manually fixed this with plugin LocoTranslate, but every time Woo updates, my fix is overwritten.

**Comment by anastas10s-afk** [comment]
5957019-zen

**Comment by Flagmans1990** [comment]
The same issue in Lithuanian language. Waiting for your fix. Thanks.

**Comment by coreymckrill** [comment]
As mentioned [here](https://github.com/woocommerce/woocommerce/issues/36490#issuecomment-1416193416), this problem happens because some locales have not yet updated the strings to include both singular and plural for this particular translation after [this change](https://github.com/woocommerce/woocommerce/pull/32026).

Here are the locales mentioned on this issue (as many as I could figure out anyway) and their current status for this string:

**Fixed** (If your locale is in this list and you still see the problem, make sure you've downloaded the latest version of the translations from WP Admin > Dashboard > Updates)
* [de_DE](https://translate.wordpress.org/projects/wp-plugins/woocommerce/stable/de/default/?filters[term]=Address&filters[term_scope]=scope_originals&filters[status]=current_or_waiting_or_fuzzy_or_untranslated&filters[with_plural]=yes&filters[user_login]=&filter=Apply+Filters&sort[by]=priority&sort[how]=desc)
* [es_ES](https://translate.wordpress.org/projects/wp-plugins/woocommerce/stable/es/default/?filters[term]=Address&filters[term_scope]=scope_originals&filters[status]=current_or_waiting_or_fuzzy_or_untranslated&filters[with_plural]=yes&filters[user_login]=&filter=Apply+Filters&sort[by]=priority&sort[how]=desc)
* [fr_FR](https://translate.wordpress.org/projects/wp-plugins/woocommerce/stable/fr/default/?filters[term]=Address&filters[term_scope]=scope_originals&filters[status]=current_or_waiting_or_fuzzy_or_untranslated&filters[with_plural]=yes&filters[user_login]=&filter=Apply+Filters&sort[by]=priority&sort[how]=desc)
* [hu](https://translate.wordpress.org/projects/wp-plugins/woocommerce/stable/hu/default/?filters[term]=Address&filters[term_scope]=scope_originals&filters[status]=current_or_waiting_or_fuzzy_or_untranslated&filters[with_plural]=yes&filters[user_login]=&filter=Apply+Filters&sort[by]=priority&sort[how]=desc)
* [it_IT](https://translate.wordpress.org/projects/wp-plugins/woocommerce/stable/it/default/?filters[term]=Address&filters[term_scope]=scope_originals&filters[status]=current_or_waiting_or_fuzzy_or_untranslated&filters[with_plural]=yes&filters[user_login]=&filter=Apply+Filters&sort[by]=priority&sort[how]=desc)
* [ja](https://translate.wordpress.org/projects/wp-plugins/woocommerce/stable/ja/default/?filters[term]=Address&filters[term_scope]=scope_originals&filters[status]=current_or_waiting_or_fuzzy_or_untranslated&filters[with_plural]=yes&filters[user_login]=&filter=Apply+Filters&sort[by]=priority&sort[how]=desc)
* [pl](https://translate.wordpress.org/projects/wp-plugins/woocommerce/stable/pl/default/?filters[term]=Address&filters[term_scope]=scope_originals&filters[status]=current_or_waiting_or_fuzzy_or_untranslated&filters[with_plural]=yes&filters[user_login]=&filter=Apply+Filters&sort[by]=priority&sort[how]=desc)
* [sl](https://translate.wordpress.org/projects/wp-plugins/woocommerce/stable/sl/default/?filters[term]=Address&filters[term_scope]=scope_originals&filters[status]=current_or_waiting_or_fuzzy_or_untranslated&filters[with_plural]=yes&filters[user_login]=&filter=Apply+Filters&sort[by]=priority&sort[how]=desc)

**Updated, but waiting for approval**
* [en_GB](https://translate.wordpress.org/projects/wp-plugins/woocommerce/stable/en-gb/default/?filters[term]=Address&filters[term_scope]=scope_originals&filters[status]=current_or_waiting_or_fuzzy_or_untranslated&filters[with_plural]=yes&filters[user_login]=&filter=Apply+Filters&sort[by]=priority&sort[how]=desc)
* [pt_BR](https://translate.wordpress.org/projects/wp-plugins/woocommerce/stable/pt-br/default/?filters[term]=Address&filters[term_scope]=scope_originals&filters[status]=current_or_waiting_or_fuzzy_or_untranslated&filters[with_plural]=yes&filters[user_login]=&filter=Apply+Filters&sort[by]=priority&sort[how]=desc)
* [tr_TR](https://translate.wordpress.org/projects/wp-plugins/woocommerce/stable/tr/default/?filters[term]=Address&filters[term_scope]=scope_originals&filters[status]=current_or_waiting_or_fuzzy_or_untranslated&filters[with_plural]=yes&filters[user_login]=&filter=Apply+Filters&sort[by]=priority&sort[how]=desc)

**No translation yet**
* [da](https://translate.wordpress.org/projects/wp-plugins/woocommerce/stable/da/default/?filters[term]=Address&filters[term_scope]=scope_originals&filters[status]=current_or_waiting_or_fuzzy_or_untranslated&filters[with_plural]=yes&filters[user_login]=&filter=Apply+Filters&sort[by]=priority&sort[how]=desc)
* [lt](https://translate.wordpress.org/projects/wp-plugins/woocommerce/stable/lt/default/?filters[term]=Address&filters[term_scope]=scope_originals&filters[status]=current_or_waiting_or_fuzzy_or_untranslated&filters[with_plural]=yes&filters[user_login]=&filter=Apply+Filters&sort[by]=priority&sort[how]=desc)
* [pt_PT](https://translate.wordpress.org/projects/wp-plugins/woocommerce/stable/pt/default/?filters[term]=Address&filters[term_scope]=scope_originals&filters[status]=current_or_waiting_or_fuzzy_or_untranslated&filters[with_plural]=yes&filters[user_login]=&filter=Apply+Filters&sort[by]=priority&sort[how]=desc)
* [ro](https://translate.wordpress.org/projects/wp-plugins/woocommerce/stable/ro/default/?filters[term]=Address&filters[term_scope]=scope_originals&filters[status]=current_or_waiting_or_fuzzy_or_untranslated&filters[with_plural]=yes&filters[user_login]=&filter=Apply+Filters&sort[by]=priority&sort[how]=desc)

If you have this issue, but your locale isn't listed above, you can check the status [here](https://translate.wordpress.org/projects/wp-plugins/woocommerce/stable/).

If you don't want to wait for your translation to get updated, you can try adding this code snippet, replacing `{your translation here}` with the appropriate translation for "Addresses". If you're not familiar or comfortable with modifying code on your site, you should probably just wait, though.

```php
// Temporary fix for missing translation.
// See https://github.com/woocommerce/woocommerce/issues/36490
add_filter(
	'ngettext_woocommerce',
	function( $translation, $single, $plural, $number ) {
		if ( 'Addresses' === $plural && $number === 2 ) {
			$translation = '{your translation here}';
		}

		return $translation;
	},
	10,
	4
);
```

**Comment by coreymckrill** [comment]
By the way, the reason the "Addresses" string is disappearing altogether when it hasn't been translated yet is because of how GlotPress (the translation engine for WordPress) exports plural forms. When the plural is missing, an empty string is generated instead of it just being omitted from the translation file, which causes WordPress to output the empty string as the translation.

Upstream issue here: https://github.com/GlotPress/GlotPress/issues/1191

**Comment by matanmn33** [comment]
> As mentioned [here](https://github.com/woocommerce/woocommerce/issues/36490#issuecomment-1416193416), this problem happens because some locales have not yet updated the strings to include both singular and plural for this particular translation after [this change](https://github.com/woocommerce/woocommerce/pull/32026).
> 
> Here are the locales mentioned on this issue (as many as I could figure out anyway) and their current status for this string:
> 
> **Fixed** (If your locale is in this list and you still see the problem, make sure you've downloaded the latest version of the translations from WP Admin > Dashboard > Updates)
> 
> * [de_DE](https://translate.wordpress.org/projects/wp-plugins/woocommerce/stable/de/default/?filters%5Bterm%5D=Address&filters%5Bterm_scope%5D=scope_originals&filters%5Bstatus%5D=current_or_waiting_or_fuzzy_or_untranslated&filters%5Bwith_plural%5D=yes&filters%5Buser_login%5D=&filter=Apply+Filters&sort%5Bby%5D=priority&sort%5Bhow%5D=desc)
> * [es_ES](https://translate.wordpress.org/projects/wp-plugins/woocommerce/stable/es/default/?filters%5Bterm%5D=Address&filters%5Bterm_scope%5D=scope_originals&filters%5Bstatus%5D=current_or_waiting_or_fuzzy_or_untranslated&filters%5Bwith_plural%5D=yes&filters%5Buser_login%5D=&filter=Apply+Filters&sort%5Bby%5D=priority&sort%5Bhow%5D=desc)
> * [fr_FR](https://translate.wordpress.org/projects/wp-plugins/woocommerce/stable/fr/default/?filters%5Bterm%5D=Address&filters%5Bterm_scope%5D=scope_originals&filters%5Bstatus%5D=current_or_waiting_or_fuzzy_or_untranslated&filters%5Bwith_plural%5D=yes&filters%5Buser_login%5D=&filter=Apply+Filters&sort%5Bby%5D=priority&sort%5Bhow%5D=desc)
> * [hu](https://translate.wordpress.org/projects/wp-plugins/woocommerce/stable/hu/default/?filters%5Bterm%5D=Address&filters%5Bterm_scope%5D=scope_originals&filters%5Bstatus%5D=current_or_waiting_or_fuzzy_or_untranslated&filters%5Bwith_plural%5D=yes&filters%5Buser_login%5D=&filter=Apply+Filters&sort%5Bby%5D=priority&sort%5Bhow%5D=desc)
> * [it_IT](https://translate.wordpress.org/projects/wp-plugins/woocommerce/stable/it/default/?filters%5Bterm%5D=Address&filters%5Bterm_scope%5D=scope_originals&filters%5Bstatus%5D=current_or_waiting_or_fuzzy_or_untranslated&filters%5Bwith_plural%5D=yes&filters%5Buser_login%5D=&filter=Apply+Filters&sort%5Bby%5D=priority&sort%5Bhow%5D=desc)
> * [ja](https://translate.wordpress.org/projects/wp-plugins/woocommerce/stable/ja/default/?filters%5Bterm%5D=Address&filters%5Bterm_scope%5D=scope_originals&filters%5Bstatus%5D=current_or_waiting_or_fuzzy_or_untranslated&filters%5Bwith_plural%5D=yes&filters%5Buser_login%5D=&filter=Apply+Filters&sort%5Bby%5D=priority&sort%5Bhow%5D=desc)
> * [pl](https://translate.wordpress.org/projects/wp-plugins/woocommerce/stable/pl/default/?filters%5Bterm%5D=Address&filters%5Bterm_scope%5D=scope_originals&filters%5Bstatus%5D=current_or_waiting_or_fuzzy_or_untranslated&filters%5Bwith_plural%5D=yes&filters%5Buser_login%5D=&filter=Apply+Filters&sort%5Bby%5D=priority&sort%5Bhow%5D=desc)
> * [sl](https://translate.wordpress.org/projects/wp-plugins/woocommerce/stable/sl/default/?filters%5Bterm%5D=Address&filters%5Bterm_scope%5D=scope_originals&filters%5Bstatus%5D=current_or_waiting_or_fuzzy_or_untranslated&filters%5Bwith_plural%5D=yes&filters%5Buser_login%5D=&filter=Apply+Filters&sort%5Bby%5D=priority&sort%5Bhow%5D=desc)
> 
> **Updated, but waiting for approval**
> 
> * [en_GB](https://translate.wordpress.org/projects/wp-plugins/woocommerce/stable/en-gb/default/?filters%5Bterm%5D=Address&filters%5Bterm_scope%5D=scope_originals&filters%5Bstatus%5D=current_or_waiting_or_fuzzy_or_untranslated&filters%5Bwith_plural%5D=yes&filters%5Buser_login%5D=&filter=Apply+Filters&sort%5Bby%5D=priority&sort%5Bhow%5D=desc)
> * [pt_BR](https://translate.wordpress.org/projects/wp-plugins/woocommerce/stable/pt-br/default/?filters%5Bterm%5D=Address&filters%5Bterm_scope%5D=scope_originals&filters%5Bstatus%5D=current_or_waiting_or_fuzzy_or_untranslated&filters%5Bwith_plural%5D=yes&filters%5Buser_login%5D=&filter=Apply+Filters&sort%5Bby%5D=priority&sort%5Bhow%5D=desc)
> * [tr_TR](https://translate.wordpress.org/projects/wp-plugins/woocommerce/stable/tr/default/?filters%5Bterm%5D=Address&filters%5Bterm_scope%5D=scope_originals&filters%5Bstatus%5D=current_or_waiting_or_fuzzy_or_untranslated&filters%5Bwith_plural%5D=yes&filters%5Buser_login%5D=&filter=Apply+Filters&sort%5Bby%5D=priority&sort%5Bhow%5D=desc)
> 
> **No translation yet**
> 
> * [da](https://translate.wordpress.org/projects/wp-plugins/woocommerce/stable/da/default/?filters%5Bterm%5D=Address&filters%5Bterm_scope%5D=scope_originals&filters%5Bstatus%5D=current_or_waiting_or_fuzzy_or_untranslated&filters%5Bwith_plural%5D=yes&filters%5Buser_login%5D=&filter=Apply+Filters&sort%5Bby%5D=priority&sort%5Bhow%5D=desc)
> * [lt](https://translate.wordpress.org/projects/wp-plugins/woocommerce/stable/lt/default/?filters%5Bterm%5D=Address&filters%5Bterm_scope%5D=scope_originals&filters%5Bstatus%5D=current_or_waiting_or_fuzzy_or_untranslated&filters%5Bwith_plural%5D=yes&filters%5Buser_login%5D=&filter=Apply+Filters&sort%5Bby%5D=priority&sort%5Bhow%5D=desc)
> * [pt_PT](https://translate.wordpress.org/projects/wp-plugins/woocommerce/stable/pt/default/?filters%5Bterm%5D=Address&filters%5Bterm_scope%5D=scope_originals&filters%5Bstatus%5D=current_or_waiting_or_fuzzy_or_untranslated&filters%5Bwith_plural%5D=yes&filters%5Buser_login%5D=&filter=Apply+Filters&sort%5Bby%5D=priority&sort%5Bhow%5D=desc)
> * [ro](https://translate.wordpress.org/projects/wp-plugins/woocommerce/stable/ro/default/?filters%5Bterm%5D=Address&filters%5Bterm_scope%5D=scope_originals&filters%5Bstatus%5D=current_or_waiting_or_fuzzy_or_untranslated&filters%5Bwith_plural%5D=yes&filters%5Buser_login%5D=&filter=Apply+Filters&sort%5Bby%5D=priority&sort%5Bhow%5D=desc)
> 
> If you have this issue, but your locale isn't listed above, you can check the status [here](https://translate.wordpress.org/projects/wp-plugins/woocommerce/stable/).
> 
> If you don't want to wait for your translation to get updated, you can try adding this code snippet, replacing `{your translation here}` with the appropriate translation for "Addresses". If you're not familiar or comfortable with modifying code on your site, you should probably just wait, though.
> 
> ```
> // Temporary fix for missing translation.
> // See https://github.com/woocommerce/woocommerce/issues/36490
> add_filter(
> 	'ngettext_woocommerce',
> 	function( $translation, $single, $plural, $number ) {
> 		if ( 'Addresses' === $plural && $number === 2 ) {
> 			$translation = '{your translation here}';
> 		}
> 
> 		return $translation;
> 	},
> 	10,
> 	4
> );
> ```

What about Hebrew language? he_IL

**Comment by munford** [comment]
I used the code fix @matanmn33 posted which worked but the "edit" button on the billing address is not linking, and I can't even click (or copy text) inside that box. The delivery address is fine. Anyone know why that might be?

**Comment by matanmn33** [comment]
> I used the code fix @matanmn33 posted which worked but the "edit" button on the billing address is not linking, and I can't even click (or copy text) inside that box. The delivery address is fine. Anyone know why that might be?

Maybe there is something to do with the last version 7.4.0 i think, i did not update from 7.2.3 because the "Addresses" bug yet and i still waiting to see the change log.
Ver.7.2.3 actually works pretty good to me.
Can you send a link to your website so we can try and check if something block it?

**Comment by rashedripon** [comment]
Reported the same 5972955-zen

**Comment by munford** [comment]
Hi the site is https://wift.nu/
thanks for taking a look.

**Comment by stoltzrx** [comment]
5986880-zen **English (UK)**

**Comment by munford** [comment]
HI @matanmn33 I fixed that issue with the edit button.

**Comment by xue28** [comment]
5984373-zen

**Comment by stoltzrx** [comment]
5978688-zen

**Comment by stoltzrx** [comment]
5987481-zen

**Comment by stoltzrx** [comment]
5979809-zen **(pt_PT)**

**Comment by xue28** [comment]
5996278-zen

**Comment by raifd** [comment]
5974368-zen

**Comment by adrianrus9** [comment]
the same problem in Romanian Language :D

**Comment by xue28** [comment]
6000638-zen

**Comment by stoltzrx** [comment]
6004039-zen (Ukrainian)

**Comment by lynnjat7** [comment]
Danish / Dansk version of the same issue: 5966686-zen

**Comment by xue28** [comment]
6010192-zen

**Comment by Babylon1999** [comment]
One more case in zd-6011483 (UK)

**Comment by stoltzrx** [comment]
6012279-zen

**Comment by Babylon1999** [comment]
one more case (German) in zd-6019001

**Comment by webthread** [comment]
Same issue for AU, I noticed on a fresh install it displayed as expected but when upgrading from 7.2.X it disappears https://github.com/woocommerce/woocommerce/issues/36972

**Comment by xue28** [comment]
5973483-zen

**Comment by avantegarde** [comment]
6010219-zen

**Comment by Uziel302** [comment]
I used the temporary fix listed above and found out it only solves the issue in personal area, in the admin's dashboard there is still a bug the admin can't edit the locations.

**Comment by coreymckrill** [comment]
@Uziel302 can you share a screenshot so I can understand what you're referring to?

**Comment by stoltzrx** [comment]
6039755-zen **English (UK)**

**Comment by Uziel302** [comment]
@coreymckrill 
in this page:
https://asafmalin.com/wp-admin/admin.php?page=wc-settings&tab=shipping
using woocommerce 7.2.3 (last good version):
<img width="1612" alt="image" src="https://user-images.githubusercontent.com/40041678/224937765-93d495d9-d9d1-486f-b54a-f43198036638.png">
using woocommerce 7.4.1:
<img width="1788" alt="image" src="https://user-images.githubusercontent.com/40041678/224939872-95cfa633-1b22-4bbf-b256-b69e954b6686.png">

**Comment by coreymckrill** [comment]
@Uziel302 I think that's an unrelated issue. Could you please open a [new bug report](https://github.com/woocommerce/woocommerce/issues/new?assignees=&labels=&template=1-bug-report.yml) and describe how to replicate the issue?

**Comment by Uziel302** [comment]
@coreymckrill 
I just checked woocommerce 7.5.0 and it doesn't reproduce, thanks.

**Comment by xue28** [comment]
6053520-zen

**Comment by jaxr** [comment]
6028173-zen

**Comment by stoltzrx** [comment]
6068811-zen

**Comment by SirRahikkala** [comment]
I have the same issue with swedish and finnish languages that it does not translate with Loco translate but the site which uses WPML does translate it correctly.

**Comment by matanmn33** [comment]
@coreymckrill Hey my friend, do we have any update about this issue? it's been a while.

**Comment by coreymckrill** [comment]
Most of the locales from my [last comment](https://github.com/woocommerce/woocommerce/issues/36490#issuecomment-1437743787) have now been updated. There are still a few that are either waiting for approval or haven't been translated yet:

* [Hebrew](https://translate.wordpress.org/projects/wp-plugins/woocommerce/stable/he/default/?filters[term]=Address&filters[term_scope]=scope_originals&filters[status]=current_or_waiting_or_fuzzy_or_untranslated&filters[with_plural]=yes&filters[user_login]=&filter=Apply+Filters&sort[by]=priority&sort[how]=desc)
* [Lithuanian](https://translate.wordpress.org/projects/wp-plugins/woocommerce/stable/lt/default/?filters%5Bterm%5D=Address&filters%5Bterm_scope%5D=scope_originals&filters%5Bstatus%5D=current_or_waiting_or_fuzzy_or_untranslated&filters%5Bwith_plural%5D=yes&filters%5Buser_login%5D=&filter=Apply+Filters&sort%5Bby%5D=priority&sort%5Bhow%5D=desc)
* [Portuguese (Brazil)](https://translate.wordpress.org/projects/wp-plugins/woocommerce/stable/pt-br/default/?filters%5Bterm%5D=Address&filters%5Bterm_scope%5D=scope_originals&filters%5Bstatus%5D=current_or_waiting_or_fuzzy_or_untranslated&filters%5Bwith_plural%5D=yes&filters%5Buser_login%5D=&filter=Apply+Filters&sort%5Bby%5D=priority&sort%5Bhow%5D=desc)
* [Turkish](https://translate.wordpress.org/projects/wp-plugins/woocommerce/stable/tr/default/?filters%5Bterm%5D=Address&filters%5Bterm_scope%5D=scope_originals&filters%5Bstatus%5D=current_or_waiting_or_fuzzy_or_untranslated&filters%5Bwith_plural%5D=yes&filters%5Buser_login%5D=&filter=Apply+Filters&sort%5Bby%5D=priority&sort%5Bhow%5D=desc)

For an immediate fix you can try the [code snippet](https://github.com/woocommerce/woocommerce/issues/36490#issuecomment-1437743787) from my last comment, or you can try asking for help in the #polyglots channel in [WordPress Slack](https://chat.wordpress.org/), to see if anyone there can expedite getting the translations approved for your locale.

**Comment by matanmn33** [comment]
> Most of the locales from my [last comment](https://github.com/woocommerce/woocommerce/issues/36490#issuecomment-1437743787) have now been updated. There are still a few that are either waiting for approval or haven't been translated yet:
> 
> * [Hebrew](https://translate.wordpress.org/projects/wp-plugins/woocommerce/stable/he/default/?filters%5Bterm%5D=Address&filters%5Bterm_scope%5D=scope_originals&filters%5Bstatus%5D=current_or_waiting_or_fuzzy_or_untranslated&filters%5Bwith_plural%5D=yes&filters%5Buser_login%5D=&filter=Apply+Filters&sort%5Bby%5D=priority&sort%5Bhow%5D=desc)
> * [Lithuanian](https://translate.wordpress.org/projects/wp-plugins/woocommerce/stable/lt/default/?filters%5Bterm%5D=Address&filters%5Bterm_scope%5D=scope_originals&filters%5Bstatus%5D=current_or_waiting_or_fuzzy_or_untranslated&filters%5Bwith_plural%5D=yes&filters%5Buser_login%5D=&filter=Apply+Filters&sort%5Bby%5D=priority&sort%5Bhow%5D=desc)
> * [Portuguese (Brazil)](https://translate.wordpress.org/projects/wp-plugins/woocommerce/stable/pt-br/default/?filters%5Bterm%5D=Address&filters%5Bterm_scope%5D=scope_originals&filters%5Bstatus%5D=current_or_waiting_or_fuzzy_or_untranslated&filters%5Bwith_plural%5D=yes&filters%5Buser_login%5D=&filter=Apply+Filters&sort%5Bby%5D=priority&sort%5Bhow%5D=desc)
> * [Turkish](https://translate.wordpress.org/projects/wp-plugins/woocommerce/stable/tr/default/?filters%5Bterm%5D=Address&filters%5Bterm_scope%5D=scope_originals&filters%5Bstatus%5D=current_or_waiting_or_fuzzy_or_untranslated&filters%5Bwith_plural%5D=yes&filters%5Buser_login%5D=&filter=Apply+Filters&sort%5Bby%5D=priority&sort%5Bhow%5D=desc)
> 
> For an immediate fix you can try the [code snippet](https://github.com/woocommerce/woocommerce/issues/36490#issuecomment-1437743787) from my last comment, or you can try asking for help in the #polyglots channel in [WordPress Slack](https://chat.wordpress.org/), to see if anyone there can expedite getting the translations approved for your locale.

As you said and commented, at first thank you very much, second- "Hebrew" was the first issue opened, i know not many can translate immediately so i checked the translations myself and found the only the "Singular" was correct and i suggested "Plural" translation, hope they will approve it.
BTW: the changes will be only when the next version of WooCommerce release or its a translation locale that does not need any of updates just sync?

**Comment by coreymckrill** [comment]
Kudos for submitting a translation!

🤔 I don't know for sure, but I _think_ the translation system looks for changes and generates new language packs every 24 hours (or maybe it's once a week). So once your translation is approved, you shouldn't have to wait for a new version of WooCommerce to come out.

**Comment by matanmn33** [comment]
> 



> Kudos for submitting a translation!
> 
> 🤔 I don't know for sure, but I _think_ the translation system looks for changes and generates new language packs every 24 hours (or maybe it's once a week). So once your translation is approved, you shouldn't have to wait for a new version of WooCommerce to come out.

Okay, thanks for the help here, really appreciate it, this information just closed my questions.

**Comment by xue28** [comment]
6077173-zen

**Comment by betayazilim** [comment]
Turkish language ahs same problem.
I edit woocommerce tranlastion with loco translate and added Adreses' translation for plural. It is fixed

**Comment by stelio-cristovao** [comment]
Hello!
In Brazilian Portuguese I already did all the translations with Loco Translate but the same problem persists, the menu does not translate, "Addresses" continues to appear in English.

Can anyone help me, please?

**Comment by stelio-cristovao** [comment]
I used the interim code and for now it solved the problem. I will wait for the final approval from Portuguese/Brazil.

Thanks coreymckrill for the help!

**Comment by xue28** [comment]
6282200-zen Turkish language

**Comment by emreerkan** [comment]
Turkish translation is approved by me a moment ago. Cheers

**Comment by RuView** [comment]
> +1
> 
> UK Translation missing plural:
> 
> #: includes/admin/class-wc-admin-setup-wizard.php:511 #: includes/wc-account-functions.php:102 #: packages/woocommerce-blocks/src/StoreApi/Schemas/V1/AbstractAddressSchema.php:37 #: packages/woocommerce-blocks/src/StoreApi/Utilities/OrderController.php:330 #: assets/client/admin/app/index.js:2 #: packages/woocommerce-blocks/build/wc-settings.js:1 msgid "Address" msgid_plural "Addresses" msgstr[0] "Address" msgstr[1] ""

RossThinkZap, thank you! It works!
I downloaded language file (woocommerce-ru_RU.po) from Admin Panel with Loco Translate and edited with Notepad++, after that uploaded back using Loco Translate - now everything is fine for Russian language.

My code:
`msgid "Address"
msgid_plural "Addresses"
msgstr[0] "Адрес"
msgstr[1] "Адреса"
msgstr[2] "Адреса"`

**Comment by xue28** [comment]
6473828- zen Portuguese

**Comment by raifd** [comment]
6528420-zd-woothemes

**Comment by freecookbr** [comment]
Portuguese language, same issue. Nobody can help to fix this issue?

**Comment by p11v** [comment]
Same here.

**Comment by freecookbr** [comment]
Add this function in your functions.php theme file.

// Bug correction on Adressess at My Account page
add_filter('woocommerce_account_menu_items', function($menu_links) {
    $menu_links['edit-address'] = _n('Address', 'Addresses', (1 + (int) wc_shipping_enabled()), 'domain');
    return $menu_links;
});

**Comment by freecookbr** [comment]
> Same here.

Add the function code above in your functions.php theme file.

**Comment by p11v** [comment]
> > Same here.
> 
> Add the function code above in your functions.php theme file.

I solved it by downloading the PO file and adding the plural manually just like @RuView did.

**Comment by doominio** [comment]
I just got this bug on the Spanish version of WooCommerce. Apparently it dates as far back as 2014 judging from some Github issues.

The "Addresses" linke in My Account is blank. If I install Loco Translate I can see that the translation exists. If I edit the translation and save it, then I get "Addresses" in English instead of a blank line.

**Comment by doominio** [comment]
> Add this function in your functions.php theme file.

Thanks. The code gets "Addresses" in English to appear, and I was able to edit the code in the fix to get it in Spanish.

**Comment by coreymckrill** [comment]
@doominio I checked the Spanish translation [here](https://translate.wordpress.org/projects/wp-plugins/woocommerce/stable/es/default/?filters%5Bterm%5D=Address&filters%5Bterm_scope%5D=scope_originals&filters%5Bstatus%5D=current_or_waiting_or_fuzzy_or_untranslated&filters%5Bwith_plural%5D=yes&filters%5Buser_login%5D=&filter=Apply+Filters&sort%5Bby%5D=priority&sort%5Bhow%5D=desc), and for some reason the plural form was deleted again, even though it was previously fixed. I asked about it in WordPress Slack [here](https://wordpress.slack.com/archives/C02RP50LK/p1702325253835629).

**Comment by lengySVK** [comment]
I have some issue with Slovak language. I have translated everything with Loco but still no changes. Official translation seems to be fine.
![image](https://github.com/woocommerce/woocommerce/assets/109914303/b65b0a5c-0642-4caf-9a67-6e472b797381)

Code 
![image](https://github.com/woocommerce/woocommerce/assets/109914303/bc9f683d-0fda-4456-91a6-b8570a3af231)
 
but still not translated like this:

![image](https://github.com/woocommerce/woocommerce/assets/109914303/cf34e4f9-eda5-4946-858c-d675782ba5e0)

Can someone advise what to do?
